documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
+
+Additional capital contributions can be voted on and approved by the
+Corporate Overlords during a plotting session.
+
+Question for Howard: how do we allow members *or* CO's to contribute
+funds directly to specific BIFFUD projects without having equity impact?
+
+
+### Allocating Resources to Projects
+The Corporate Overlords can vote to assign budgets to BIFFUD Projects
+which are then overseen by the project owners.  The project owners
+must select a single person to be the project's "Cashier" who is
+responsible for approving any invoices associated with a project or
+working with the BIFFUD Treasurer to make purchases using BIFFUD funds.
+
+At the time of budget approval the Corporate Overlords must specify if
+the resources are being "given" or "loaned" to the project.
+
+Invoices that are not approved by a project's Cashier will not be compensated
+unless they are explicitly approved by the BIFFUD Treasurer.
+
+
+### Distributing Profits
+Profits are to be calculated at the end of the fiscal year.  If by
+some miracle there are profits on a given year, the following formula
+will dictate its distribution
+
+1. A mandatory distribution of profit will be made to offset individual
+taxes on that profit.  Corporate Overlords may choose to prevent this
+distribution my making a 70% vote during a plotting session.
+
+2. 60% of the remaining funds will be distributed to the projects
+in proportion to the project-level profit distribution.
+
+2b. If a project has a profit allocation and the project had been allocated
+a "loan" from BIFFUD, that project's allocated profit is reduced by BIFFUD
+and subtracted from the outstanding balance.
+
+2c. The project owners may vote to decide to either distribute the profits
+proportionally according to their percentage ownership of the project, or
+to allocate the profit towards future project expenses.
+
+3. The remaining 40% of the profits is kept by BIFFUD.
```

Comment:
I personally don't think it should be pro rated -- I'm not sure it would be contentious to only get the % of the time you were involved, that seems fair to me.